Jenkinsfile-testing

```diff
@@ -0,0 +1,63 @@
+library("tdr-jenkinslib")
```

Comment:
Maybe this should be renamed `Jenkinsfile-test` to make it consistent with the other Jenkins test jobs in other repos?

Reply:
I copied it off the frontend but that's so old now it's not a good example. I've renamed it.